doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -614,148 +619,101 @@ syntax: :n:`let fix @ident @binders := @term in` stands for
-      :cmd:`Import` and its variants. You have to explicitly give their fully
-      qualified name to refer to them.
+   .. prodn::
+      assumption_token ::= {| Axiom | Axioms }
+      | {| Conjecture | Conjectures }
+      | {| Parameter | Parameters }
+      | {| Hypothesis | Hypotheses }
+      | {| Variable | Variables }
+      assumpt ::= {+ @ident_decl } @of_type
+      ident_decl ::= @ident {? @univ_decl }
+      of_type ::= {| : | :> | :>> } @term
 
-   .. cmdv:: {? Local } Parameters {+ ( {+ @ident } : @type ) }
-             {? Local } Axiom {+ ( {+ @ident } : @type ) }
-             {? Local } Axioms {+ ( {+ @ident } : @type ) }
-             {? Local } Conjecture {+ ( {+ @ident } : @type ) }
-             {? Local } Conjectures {+ ( {+ @ident } : @type ) }
-      :name: Parameters; Axiom; Axioms; Conjecture; Conjectures
+   This command binds one or more :n:`@ident`\s to specified :n:`@type`\(s) as their specifications in
+   the global context. The fact asserted by the :n:`@type` is accepted as a
+   postulate.
 
-      These variants are synonyms of :n:`{? Local } Parameter {+ ( {+ @ident } : @type ) }`.
+   :cmd:`Axiom`, :cmd:`Conjecture`, :cmd:`Parameter` and their plural forms
+   are equivalent.  They can take the :attr:`local` attribute (see :ref:`gallina-attributes`),
+   which makes the defined :n:`@ident`\s accessible by :cmd:`Import` and its variants
+   only through their fully qualified names.
 
-   .. cmdv:: Variable  {+ ( {+ @ident } : @type ) }
-             Variables {+ ( {+ @ident } : @type ) }
-             Hypothesis {+ ( {+ @ident } : @type ) }
-             Hypotheses {+ ( {+ @ident } : @type ) }
-      :name: Variable (outside a section); Variables (outside a section); Hypothesis (outside a section); Hypotheses (outside a section)
+   Similarly, :cmd:`Hypothesis`, :cmd:`Variable` and their plural forms are equivalent.  Outside
+   of a section, these are equivalent to :n:`Local Parameter`.  Inside a section, the
+   :n:`@ident`\s defined are only accessible within the section.  When the current section
+   is closed, the :n:`@ident`\s
+   becomes undefined and every object using this variable will be explicitly
+   parameterized (i.e., the variable is *discharged*).  See Section :ref:`section-mechanism`.
 
-      Outside of any section, these variants are synonyms of
-      :n:`Local Parameter {+ ( {+ @ident } : @type ) }`.
-      For their meaning inside a section, see :cmd:`Variable` in
-      :ref:`section-mechanism`.
+   .. exn:: @ident already exists.
+      :name: @ident already exists. (Axiom)
+      :undocumented:
 
-      .. warn:: @ident is declared as a local axiom [local-declaration,scope]
+   .. warn:: @ident is declared as a local axiom [local-declaration,scope]
 
-         Warning generated when using :cmd:`Variable` instead of
-         :cmd:`Local Parameter`.
+      Warning generated when using :cmd:`Variable` instead of
```

Comment:
```suggestion
      Warning generated when using :cmd:`Variable`, :cmd:`Hypothesis` or their plural forms instead of
```

Reply:
How about this instead?

Warning generated when using :cmd:`Variable` or its equivalents
      instead of :n:`Local Parameter` or its equivalents.

Reply:
Sure, that sounds good too.